libs/ui/packages/native/src/components/Layout/Divider/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { Flex } from "../index";
+import styled from "styled-components/native";
+
+const Divider = styled(Flex).attrs(() => ({
```

Comment:
here you can pass directly the object as param of `attrs` if you have no use for the function:

```typescript
const Divider = styled(Flex).attrs({
  my: 4,
  height: 1,
  backgroundColor: "neutral.c40",
})``;